document_quick_access/README.rst

```diff
@@ -0,0 +1,89 @@
+=====================
+Document Quick Access
+=====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--ux-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-ux/tree/11.0/document_quick_access
+    :alt: OCA/server-ux
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-ux-11-0/server-ux-11-0-document_quick_access
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/250/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to search any kind of records through a launcher.
+With this, we can add a QR in our reports in order to search elements faster.
+It could be used to add this QR (on a label) on external documents.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+# Access on developer mode
+# Access `Settings > Technical > Document Quick Access`
+# Create a record selecting the model, format and priority
+
+Usage
+=====
+
+# Click on the QR button on the menu
```

Comment:
I'm wondering if this can't be added as a general listener in whole web client for not needing to click in any place. Would it be an improvement? If not, I suppose this button is very accessible from any place. Isn't it?

Reply:
It could be interesting, but this could blocked by the standard barcode reader of odoo. 
Maybe, we could add a shortcut? Right now, there is a button on the menu accessible everywhere.

Reply:
OK, put it on the roadmap the other option and specify here in the README where the button is in the interface (`upper right in the navbar` for example).

Reply:
OK! Added the roadmap and regenerated the readme.
I have been reviewing this, and the shortcut is easier to add if we use `web_responsive`. Harder with the standard odoo.

Reply:
I added a shortcut Alt + Shift + Q